packages/rest/test/unit/parser.unit.ts

```diff
@@ -54,11 +57,51 @@ describe('operationArgsParser', () => {
     });
     const route = givenResolvedRoute(spec);
 
-    const args = await parseOperationArgs(req, route);
+    const args = await parseOperationArgs(req, route, requestBodyParser);
 
     expect(args).to.eql([{key: 'value'}]);
   });
 
+  it('parses body string as json', async () => {
```

Comment:
This is effectively an integration test, because you are checking how `parseOperationArgs` integrates with `requestBodyParser`. Please move it to `test/integration` directory.

Please review your overall testing strategy and decide what should be covered at what level. For example, I think the next test "parses body number as json" does not need to go through `parseOperationArgs` at all, it should be enough to test this at requestBodyParser level, or even in a unit-test for JsonBodyParser.

Reply:
Refactored as body-parser unit tests